test/unit/models/testRunnerConfiguration/TestPreviewerConfigurationServiceTest.php

```diff
@@ -0,0 +1,111 @@
+<?php
+
+/**
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; under version 2
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2020 (original work) Open Assessment Technologies SA
+ */
+
+declare(strict_types=1);
+
+namespace unit\models\testRunnerConfiguration;
+
+use oat\generis\test\TestCase;
+use oat\tao\model\providers\ProviderModule;
+use oat\taoQtiTest\models\runner\config\QtiRunnerConfig;
+use oat\taoQtiTestPreviewer\models\testRunnerConfiguration\TestPreviewerConfigObject;
+use oat\taoQtiTestPreviewer\models\testRunnerConfiguration\TestPreviewerConfigurationService;
+use oat\taoTests\models\runner\plugins\TestPlugin;
+use oat\taoTests\models\runner\plugins\TestPluginService;
+use oat\taoTests\models\runner\providers\TestProviderService;
+
+class TestPreviewerConfigurationServiceTest extends TestCase
+{
+
+    /**
+     * @var TestPreviewerConfigurationService
+     */
+    private $subject;
+
+    private const OPTIONS = ['c' => 'd'];
```

Comment:
Do you think this constant would be better declared in other part of this class?

Reply:
To add my 2 cents, I would always go with constants first, and vars after. Not sure if there is a standard defined for this though.

Reply:
Yes, that is the standard. I just wanted to let him discover by himself :)